lib/test_changes/config_setup_service.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_changes/inferred_config'
+
+module TestChanges
+  module ConfigSetupService
+    def self.call
+      config = Config.new('.test_changes.yaml')
+
+      return config if config.exists?
+
+      if File.exist?('./config/application.rb')
+        return use_rspec_rails('./bin/rspec') if File.exist?('./bin/rspec')
+        return use_rspec_rails('bundle exec rspec') if File.directory?('./spec')
+        return use_testunit_rails('bundle exec ruby -Itest') if File.directory?('./test')
+      end
+
+      fail TestChanges::Error, "No .test_changes.yaml found"
+    end
+
+    private
+
+    def self.use_rspec_rails(bin)
+      InferredConfig.new(
+        test_tool_command: bin,
+        project_type_name: 'rspec_rails',
+        finding_patterns_map: {
+          '^app/(models)/(.+).rb' => 'spec/\1/\2_spec.rb',
+          '^app/(controller|helper|view)s/(.+).rb' => 'spec/controllers/\2_\1_spec.rb'
+        }
+      )
+    end
+
+    def self.use_testunit_rails(bin)
+      InferredConfig.new(
```

Comment:
Hehe. I think it's okay to keep the class because Config and InferredConfig have different dependencies. Config gets it data from `.test_changes.yaml` while InferredConfig is used internally. I considered converting `InferredConfig` to a Struct but maybe later.